README.md

```diff
@@ -37,6 +37,41 @@ url.host                # => "www.twingly.co.uk"
 url.origin              # => "http://www.twingly.co.uk"
 url.path                # => "/search"
 url.without_scheme      # => "//www.twingly.co.uk/search"
+url.userinfo            # => ""
+url.user                # => ""
+url.password            # => ""
+url.valid?              # => "true"
+
+url = Twingly::URL.parse("http://räksmörgås.макдональдс.рф/foo")
+url.scheme              # => "http"
+url.trd                 # => "räksmörgås"
+url.sld                 # => "макдональдс"
+url.tld                 # => "рф"
+url.ttld                # => "рф"
+url.domain              # => "макдональдс.рф"
+url.host                # => "räksmörgås.макдональдс.рф"
+url.origin              # => "http://xn--rksmrgs-5wao1o.xn--80aalb1aicli8a5i.xn--p1ai"
+url.path                # => "/foo"
+url.without_scheme      # => "//räksmörgås.макдональдс.рф/foo"
```

Comment:
How about we print the normalized version as well (for all parts).

```
url.without_scheme
url.normalized.without_scheme
```

Since they return different strings.